var/spack/repos/builtin/packages/py-biom-format/package.py

```diff
@@ -27,4 +29,4 @@ class PyBiomFormat(PythonPackage):
     depends_on('py-scipy@0.13.0:', type=('build', 'run'))
     depends_on('py-pandas@0.19.2:', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('py-pandas@0.20.0:', type=('build', 'run'))
```

Reply:
Version 2.1.6 requires pandas >= 0.19.2, should I always default to the latest version if possible?